### ISSUE TITLE
 Validator CLI `operator-cli status` error `TypeError: Cannot read properties of null (reading 'account')`

### ISSUE THREAD

**Issue by LorisYanis** [Shardeum/shardeum-bug-reporting#343]
When I execute "operator-cli status" instead information about my node I get the next:

![555](https://user-images.githubusercontent.com/115280526/233096202-06b92cbc-f9f5-4161-8c9d-4986f3ba371a.png)

Here is the full error:
```
/home/node/app/cli/build/src/utils/fetch-network-data.js:104
    const initialParams = await fetchDataFromNetwork(config, `/account/${exports.networkAccount}?type=5`, data => data.account === null);
                                                                                                                       ^

TypeError: Cannot read properties of null (reading 'account')
    at /home/node/app/cli/build/src/utils/fetch-network-data.js:104:120
    at fetchDataFromNetwork (/home/node/app/cli/build/src/utils/fetch-network-data.js:54:13)
    at async fetchInitialParameters (/home/node/app/cli/build/src/utils/fetch-network-data.js:104:27)
    at async getAccountInfoParams (/home/node/app/cli/build/src/utils/fetch-network-data.js:194:54)
    at async Promise.all (index 4)
    at async /home/node/app/cli/build/src/node-commands.js:145:153

```

**Comment by Glitch18** [comment]
Can you post the output of `cat cli/build/active-node.json`?

Please also provide the output of 
```
cd cli
git log --oneline | head -n 1
```

**Comment by subwayaddicted** [comment]
Having the same issue

cat cli/build/active-node.json:
`{"id":"ea5c560ff1c16e390c595beb9c3601b10b4446fc93a724ada90c08c3a187d4df","ip":"43.134.188.51","port":9001,"publicKey":"9fdfb14921f062f94edc96fe73694d39d9d648e6bfc7509a2466a8a3ecdfee6a"}`

git log --oneline | head -n 1:
`f6ddf53 build: bump version to 0.1.2`

**Comment by LorisYanis** [comment]
> git log --oneline | head -n 1

Sorry for the long answer, this is the message I got:

```
~/app/cli$ git log --oneline | head -n 1
f6ddf53 build: bump version to 0.1.2
```

**Comment by swarna1101** [comment]
update to the latest node its solved in latest version 1.9.5.
